packages/airnode-deployer/src/utils/logger.ts

```diff
@@ -23,19 +41,39 @@ export function getSpinner() {
   return spinner;
 }
 
+export function useSpinner(method: OraMethod, text?: string, options?: LoggerOptions) {
+  if (text) writeLog(text, options);
+  getSpinner()[method](text);
+}
```

Comment:
This wrapper is for using a particular instance of the spinner (mainly in `infrastructure/index.ts`) while allowing us to keep writing the logs in one place.